physlr/physlr.py

```diff
@@ -2109,6 +2109,51 @@ def chr_int(x):
             return None
         return int(x[3:]) if x.startswith("chr") else int(x)
 
+    def physlr_annotate_graph_bxtochr(self):
+        """
+        Annotate a graph with a file of barcode to chromosome mappings.
+        The mapping file includes two values per line separated by tab: barcode an chromosome.
+        Usage: physlr physlr-annotate-graph-bxtochr GRAPH PATH MAPPINGS_FILE... >ANNOTATED-GRAPHVIZ
```

Comment:
Your usage command below says that only the graph and mappings file are required? So no PATH?
```suggestion
        Usage: physlr physlr-annotate-graph-bxtochr GRAPH MAPPINGS_FILE... >ANNOTATED-GRAPHVIZ
```